### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by Maplatore** [typst/typst#2598] [Ideas]

**Comment by CGMossa** [comment]
Typst is a good name. It is a name. I don't want it to change. I don't like it to change. I don't see a benefit from it changing. Please, don't take offense.

**Comment by ludwig-austermann** [comment]
I actually find English rather difficult to pronounce. If we really wanted to accommodate as many people as possible, I suggest a Spanish (most natively spoken language after Mandarin) sounding variation of the name, aka typista or typisto?. All native English, actually all indogermanic languages, speakers can easily pronounce this and we would obtain much higher comfortability metrics measured on the whole population.

But fun aside, like you mentioned, English speakers do pronounce it as it is intended to, hence only the spelling might be problematic. I think that the spelling is also not that hard, as the only rule of thumb is to remove the i in typist - think what is the easiest / most natural way to shorten that word.

Here a short list of popular & succesful software with strange names:
- MS Excel: how to spell and probably also how to pronounce,
- MS PowerPoint: how to pronounce Power,
- (La)TeX: everything :),
- OCaml: why not OhCamel or chameau,
- SQL: nobody pronounces it as intended

I think having a distinctive and non-standard name is actually important for people to remember the name (and tool) and it enables better promotion to others, as they immediately notice your software being used. Then again, it is actually rather easy to pronounce and not hard to spell.

**Comment by Enivex** [comment]
By the way: Making your first interaction with a community a manifesto about how they need to change their name comes off as a tad confrontational.

**Comment by reknih** [comment]
Thank you for this post! We are aware that there are some pronunciation issues. However, we also like our current brand. Furthermore, changing the name now would cost us a lot of money and some of the existing name recognition. We would rather spend these resources on making Typst better.
